homeassistant/components/greeneye_monitor/sensor.py

```
@@ -45,11 +45,14 @@
 
 async def async_setup_platform(
     hass: HomeAssistant,
-    config: Config,
+    config: ConfigType,
     async_add_entities: AddEntitiesCallback,
-    discovery_info: DiscoveryInfoType,
+    discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up a single GEM temperature sensor."""
+    if not discovery_info:
+        return
```

Similar to #63782
Should we raise an error? or just return?